install.rst

```diff
@@ -201,6 +202,51 @@ To install these, you can use:
   ``swig``, ``swig-python``, ``openssl``, ``bison``, and ``flex`` packages
   provide the required dependencies.
 
+* Windows
+
+  Compiling on Windows requires the installation of a development environment.
+  Zeek currently builds on Visual Studio 2019, and you can either install the
```

Comment:
I'm a n00b here — perhaps a link to an installation entry point would help? Feel free to ignore if that's too obvious for Windows folk.